### PR TITLE
Featire/istio

### PR DESCRIPTION
**Pull request by takama**
renew istio

### CODE REVIEW

File: roles/istio/README.md [hunk 1]
```diff
@@ -62,6 +62,11 @@ Master hosts nsmes
 k8s_master_hosts: {}
 ```
 
+Node hosts nsmes
```
Comment (STASiAN) [inline roles/istio/README.md]: maybe names!?

File: roles/istio/defaults/main.yml [hunk 2]
```diff
@@ -30,13 +30,20 @@ tmp_dir: /tmp
 # Master hosts nsmes
```
Comment (STASiAN) [inline roles/istio/defaults/main.yml]: names here

**Review by STASiAN — APPROVED** [review]

**Review by vsaveliev — APPROVED** [review]
LGTM

**Review by rumyantseva — APPROVED** [review]
LGTM

File: roles/istio/README.md (hunk 1 above)
Comment (takama) [reply roles/istio/README.md]: correct: names

File: roles/istio/defaults/main.yml (hunk 2 above)
Comment (takama) [reply roles/istio/defaults/main.yml]: thanks

**Review by IKSIN — APPROVED** [review]

**Pull request merged**